doc/releases/changelog-dev.md

```diff
@@ -351,10 +351,13 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
   which were transformed using `qml.transform.insert()`.
   [(#2857)](https://github.com/PennyLaneAI/pennylane/pull/2857) 
 
+* Add GitHub Actions workflow to enable GPU based unittests to run.
+  [(#2862)](https://github.com/PennyLaneAI/pennylane/pull/2862)
```

Comment:
It should be okay to have no item in a changelog section because this change would be "invisible" to users. Great to add yourself to the list of contributors though! :)

Reply:
Oh True I see what you mean, no I'll remove this update since technically this proposes no changes to pennylane itself

Reply:
Updated